cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -598,9 +610,23 @@ impl ClientTrait for CatalogClient {
 
     async fn create_catalog(
         &self,
-        _catalog_name: impl AsRef<str> + Send + Sync,
-    ) -> Result<(), CatalogClientError> {
-        Ok(())
+        catalog_name: impl AsRef<str> + Send + Sync,
+    ) -> Result<PublishResponse, CatalogClientError> {
+        let catalog = str_to_catalog_name(catalog_name)?;
+        let package = str_to_package_name("unused")?;
```

Comment:
(follow-up) I'm not clear at what point this might break, for instance this endpoint depends on a corresponding `create_package(name)` to be called first, in part because I don't entirely understand the rationale for https://github.com/flox/catalog-server/pull/285